orga/tests/integration/components/campaign/update-form_test.js

```diff
@@ -35,20 +23,34 @@ module('Integration | Component | Campaign::UpdateForm', function (hooks) {
     assert.dom(screen.getByText('indique un champ obligatoire')).exists();
   });
 
-  test('it should send campaign update action when submitted', async function (assert) {
-    // when
-    await renderScreen(
-      hbs`<Campaign::UpdateForm @campaign={{this.campaign}} @onSubmit={{this.updateCampaignSpy}} @onCancel={{this.cancelSpy}} />`
-    );
+  module('When campaign type is ASSESSMENT', function () {
```

Comment:
**question** : pourquoi ce contexte est remonté ? normalement la seule différence entre le type de campagne c'est la présence ou l'absence de titre

Reply:
Je voulais re organiser le fichier de test en fonction de type de campagne , avant dans le teste de " it should contain inputs, attributes and validation button " on ne teste pas si le titre existe ou pas, j'ai vu dans les autres tests qu'on teste seulement quand le titre n'existe pas mais pas quand il est la, du coupe ou lieu de refaire un teste juste pour le titre,  je l'ai ajouter dans le même teste qui verifier les autres attributes.
c'est pourquoi j'ai bougé ce teste dans le context de when `isTypeassessment = true`